### DISCUSSION TITLE
Why do the python download commands run in quite mode?

### DISCUSSION THREAD

**Discussion by SBaori** [pyenv/pyenv#2841] [Ideas] [answered]
So I was looking through the python-build scripts and noticed that the curl and wget (theres one more but i have limited knowledge of it) run in quiet (less verbose) mode hence disabling the download progress bar. Is there any specific reason for that? The download is anyway being written to a file, so why not also show the progress bar?

**Comment by SBaori** [comment]
Okay, just found out about the "-v" flag. But for curl it still does not show the progress bar.

**Reply by Aimen-prog** [reply]
If you want to show the progress bar while using curl, you typically use the -# option. This option displays a progress bar along with additional information
example:
curl -# -0 https://example.com/file.zip

**Reply by SBaori** [reply]
Should this added into the install scripts which gets enabled in the verbose mode? I can create a PR.

**Comment by native-api** [comment, marked as answer]
We run `curl` (and other downloaders) without a progress bar because a progress bar only works when writing to a terminal (because it works by rapidly rewriting the screen). While we always pipe `curl`'s output to the build log file.
